docs/introduction/release_process.md

```diff
@@ -25,7 +25,7 @@ They can include:
 
 ## Patch release
 
-Patch releases occur every two weeks. They are also backward compatible (source compatible). The last digit in the release number indicates the patch release. In Mbed OS 5.2.3, `3` shows the patch release is the third release in Mbed 5.2.
+Patch releases occur on the third Wednesday of every month. They are also backward compatible (source compatible). The last digit in the release number indicates the patch release. In Mbed OS 5.2.3, `3` shows the patch release is the third release in Mbed 5.2.
```

Comment:
Not strictly true. From January onwards we will make 'a' release on the 3rd Wednesday of every month. This could be a patch, feature or major release depending on what has landed on Master since the last release.

Reply:
I thought we should wait until January to change that in the docs and that for now we should indicate what the current release cadence is. What do you think?

Reply:
Well you could update it now and then have to update it again in 2 months time :)

Reply:
I'm fine with that too